specification/cognitiveservices/OpenAI.Inference/models/azure.cognitive.search.options.tsp

```diff
@@ -101,4 +109,7 @@ model AzureCognitiveSearchChatExtensionConfiguration {
 
   @doc("When using embeddings, specifies the API key to use with the provided embeddings endpoint.")
   embeddingKey?: string;
+
+  @doc("The embedding model deployment name in the same Azure OpenAI resource. Used for vector search. Should be used exclusively with embeddingEndpoint and embeddingKey. When embeddingDeploymentName parameter is provided, the service will evaluate embedding model directly internally, and no additional authentication information is needed. If embeddingDeploymentName is not provided, the service will use the provided embeddingEndpoint and embeddingKey to retrieve embeddings from the specified endpoint, and the embeddingKey will be used as the authentication information, and the AAD is not supported to access the external embeddingEndpoint, and if the external Azure OpenAI service has network restriction, the external API call will fail.")
```

Comment:
I'm amused this doc is mostly generated by Copilot and I only add some points and made some factual correction. I know it is too verbose for doc.. Will try to summarize it.